.travis.yml

```diff
@@ -16,13 +24,12 @@ before_install:
   - source activate myenv
   - pip install --upgrade pip
 install:
-  - pip install -r requirements.txt
   - pip install pytest pytest-cov coveralls
   - pip install flake8 black
-  - pip install .[all]
   - pip install mypy
+  - pip install -e .$DEP
```

Comment:
This seems to be a common, quite mysterious problem, see here e.g https://github.com/pytest-dev/pytest-cov/issues/98

I solved this now by doing `cd test` before executing pytest. I also changed the order to avoid having to `cd ..` after the tests and also because `flake8`, `black`, and `mypy` are much faster than the tests, so we can first check for that.